<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nuevoExperimento</class>
 <widget class="QDialog" name="nuevoExperimento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="descripcion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripcion</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="comentario">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Comenterio</string>
   </property>
  </widget>
  <widget class="QSlider" name="scrollParametrosValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="parametroValue">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Información</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Parámentos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Usuario">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="unidadValue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="unidadSelection">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Unidad</string>
   </property>
   <widget class="QRadioButton" name="unidadPorcentaje">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Porcentaje</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="unidadCarga">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Carga</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="unidadTension">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tension</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnComenzar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Comenzar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel de carga</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
